src/components/StopPointsEditor/Generic/GenericStopPointEditor.tsx

```diff
@@ -20,6 +20,10 @@ import {
 } from '../common/FrontTextTextField';
 import { QuayRefField, useOnQuayRefChange } from '../common/QuayRefField';
 import { StopPointEditorProps } from '../common/StopPointEditorProps';
+import { PositionIcon } from '@entur/icons';
+import '../styles.scss';
```

Comment:
I think this stylesheet is imported elsewhere (I would expect it to be) so it's better to create a new stylesheet file in the same folder, and include it here?